File: alan.cpp
int op(int a, int b){
  return a + b;
}

int parse(int a) {
  return a + 1;
}

int somar(int a, int b){
  a = parse(a);
  b = parse(b);
  return op(a, b);
}

int main(){
  return 0;
}

File: alan.py
def op(a, b):
    return a + b


def parse(a):
    return a + 1


def somar(a, b):
    a = parse(a)
    b = parse(b)

    return op(a, b)


def __main__():
    print(somar(1, 2))
